azure-samples/pom.xml

```diff
@@ -87,6 +87,16 @@
             <groupId>com.azure</groupId>
             <artifactId>azure-storage-blob</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-documentdb</artifactId>
```

Comment:
Please take a look whether you can use latest cosmos (not sure, azure-cosmos 4.0.0?).

Reply:
done, version is 4.0.1-beta.1